interp/constrexpr_ops.ml

```diff
@@ -45,8 +45,11 @@ let names_of_local_binders bl =
 (**********************************************************************)
 (* Functions on constr_expr *)
 
+(* Note: redundant Numeral representations such as -0 and +0 (or different
+   numbers of leading zeros) are considered different here. *)
```

Comment:
Caring about differences in the number of leading zeros breaks compatibility with v8.6 and trunk, right?